docs/en-US/component/collapse.md

```
@@ -83,6 +92,7 @@ collapse/prevent-collapsing
 | accordion                      | whether to activate accordion mode                                                                                                                   | ^[boolean]                                     | false   |
 | expand-icon-position ^(2.9.10) | set expand icon position                                                                                                                             | ^[enum]`'left' \| 'right' `                    | right   |
 | before-collapse ^(2.9.11)      | before-collapse hook before the collapse state changes. If `false` is returned or a `Promise` is returned and then is rejected, will stop collapsing | ^[Function]`() => Promise<boolean> \| boolean` | —       |
+| collapsible                    | Specify how to trigger Collapse. Either by clicking icon or by clicking any area in header or disable collapse functionality itself                  | ^[enum]`'icon' \| 'header' \| 'disabled' `     | -       |
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)